recipes/ghcfilesystem/all/conanfile.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class GhcFilesystemRecipe(ConanFile):
+    name = "ghcfilesystem"
```

Comment:
To match the docs and other recipes should this be `ghc-filesystem`

https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#should-reference-names-use---or-_